pkg/operator/mgr/mgr.go

```diff
@@ -89,6 +104,31 @@ func (c *Cluster) Start() error {
 	return nil
 }
 
+func (c *Cluster) makeService(name string) *v1.Service {
```

Comment:
will our existing teardown/cleanup instructions cover this dynamically created resource as well or do those need to be updated?

Reply:
@jbw976 The `kubernetes.md` teardown/cleanup should suffice because it deletes the namespace. Is that good or should a line for deleting the service be added?